llarp/path/path.cpp

```diff
@@ -107,8 +107,33 @@ namespace llarp::path
   Path::send_path_control_message(
       std::string method, std::string body, std::function<void(oxen::quic::message m)> func)
   {
+    oxenc::bt_dict_producer btdp;
+    btdp.append("METHOD", method);
+    btdp.append("BODY", body);
+    auto payload = std::move(btdp).str();
```

Comment:
Might be good to wrap the BTDP in braces before you std move it out of scope like you did in the other logic